src/pykeen/nn/emb.py

```diff
@@ -170,3 +170,18 @@ def forward(
         if self.normalizer is not None:
             x = self.normalizer(x)
         return x
+
+    def get_in_canonical_shape(
+        self,
+        indicies: Optional[torch.LongTensor] = None,
```

Comment:
@mberr up to you if you like `ind` or `indicies` better. The `forward()` function uses `indicies` so I chose this to match.

Reply:
torch seems to prefer `index`, e.g. https://pytorch.org/docs/stable/generated/torch.index_select.html, so maybe we should go with that.

Reply:
for numpy, it is `indices`, e.g. https://numpy.org/doc/stable/reference/routines.indexing.html

Reply:
indic**i**es is a typo, right?

Reply:
hahaha not sure if I did this or copied it. let's also change all of them to index in this PR as well

Reply:
In b091829a I went by hand and fixed all different variations of ind, index, indices, etc. to make them all torch style.

Reply:
In the example for index_select they use `indices` as variable name again 😅

Reply:
GAHH https://pytorch.org/docs/stable/generated/torch.index_select.html

okay then what should we do? Indices since that's how it's always showing up in the docs?

Reply:
I personally think `indices` is more natural (since usually we provide more than one index)

Reply:
okay i will change it to that then... thanks for being patient